.travis.yml

```diff
@@ -7,8 +7,9 @@ matrix:
     - go: 1.7
     - go: 1.8
     - go: 1.9
+    - go: "1.10"
```

Comment:
Mind making all of them strings, for consistency?

Alternatively, you might change all of them to use ".x" suffix (so yaml parser will interpret all of them as strings). E.g. 1.7.x, 1.8.x, etc. Travis understands this suffix (and it will maintain same semantics as current: latest point release for request major.minor version).